ipatests/test_integration/test_caless.py

```diff
@@ -1574,10 +1574,11 @@ def test_server_ipa_ca_install(self):
 
         ca_master = tasks.install_ca(self.master)
         assert ca_master.returncode == 0
-        cert_update_master = self.master.run_command(['ipa-certupdate'])
-        assert cert_update_master.returncode == 0
-        cert_update_replica = self.replicas[0].run_command(['ipa-certupdate'])
-        assert cert_update_replica.returncode == 0
+        # ipa-certupdate command is not necessary anymore.
```

Comment:
This comment is unfortunately very confusing. If you're browsing the code, you think - what do you mean anymore? Was there a `ipa-certupdate` call before? Why was it there?
Please, replace it with something like "we're not calling ipa-certupdate on replica here since the next step installs CA clone there" and remove the pagure ticket reference.

Reply:
oki